src/services/email.service.js

```diff
@@ -0,0 +1,53 @@
+const nodemailer = require('nodemailer');
+
+const mailPort = process.env.MAIL_PORT;
+const host = process.env.MAIL_HOST;
+const hostEmail = process.env.HOST_EMAIL;
+const hostID = process.env.HOST_ID;
+const password = process.env.HOST_PASSWORD;
+
+const transporter = nodemailer.createTransport({
+    service: "naver",
+    host: host,
+    port: mailPort,
+    auth: {
+        user: hostID,
+        pass: password
+    }
+})
+
+require('dotenv').config()
+
+const sendResetLink = ({ resetToken, email }) => {
+    let link = `http://localhost:3000/api/users/resetPassword/emailVerify/${resetToken}`;
+    let html =
+        `<p>비밀번호 변경을 요청하셨습니다. 다음 링크를 눌러 비밀번호를 변경하기를 요청드립니다. <p>
+         <a href="${link}">${link}</a>`;
```

Comment:
## Future Suggestion
The email that is being sent now looks too plain and therefore is categorized as a spam mail on the client's side.
When I have time, this will be changed as well.